v2/emailpassword/common-customizations/sessions/token-transfer-method.mdx

```diff
@@ -0,0 +1,314 @@
+---
+id: token-transfer-method
+title: Token transfer methods
+hide_title: true
+show_ui_switcher: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/common-customizations/sessions/token-transfer-method.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+import NpmOrScriptTabs from "/src/components/tabs/NpmOrScriptTabs"
+import AngularUIImplementation from "/src/components/reusableSnippets/angularUIImplementation"
+import {Answer}from "/src/components/question"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs"
+import FrontendCustomUITabs from "/src/components/tabs/FrontendCustomUITabs"
+import FrontendMobileSubTabs from "/src/components/tabs/FrontendMobileSubTabs"
+
+<!-- TODO: re-think title -->
+# Token transfer methods
```

Comment:
How about "Switching between cookies and header based sessions"

Reply:
That seems a bit too long for the sidebar

Reply:
It wraps the text on the side bar, so it should be fine